### DISCUSSION TITLE
Welcome to register Discussions!

### DISCUSSION THREAD

**Discussion by phenax** [is-a-dev/register#442] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and suggestions.
  * Engage with other community members.
  * Get help with setting up your sub-domain

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ConnorDoesDev** [comment]
how long does it approximately take for domains/pr's to get reviewed?

**Reply by phenax** [reply]
I try to clear prs once every day but it may take upto 2 days for a pr to get merged and deployed

**Reply by ConnorDoesDev** [reply]
alright! thank you. I'm sorry for my dumbness lol
I fixed my 'accident' with the CNAME.

**Reply by ConnorDoesDev** [reply]
thanks for 'accepting' the PR! 
You're very quick

**Reply by phenax** [reply]
That's alright. 😄. Your cname record has been published. You can make the [required changes to your repo settings](https://github.com/is-a-dev/register/blob/main/docs/hosted-at/github-pages.md#configuring-your-repo) and it'll be good to go.

**Comment by ConnorDoesDev** [comment]
Hello, @phenax , I just realized I have a domain here already made for my bot which I never used and has the wrong credentials and CNAME etc
Could I still use it? 
It's zerotwo.json

**Reply by ConnorDoesDev** [reply]
ok, nevermind, works. [Here](https://zerotwo.is-a.dev)

**Comment by ConnorDoesDev** [comment]
Hi! Am I able to use gitbook with is-a.dev?

**Reply by phenax** [reply]
Yes. [Ref to gitbook docs](https://docs.gitbook.com/hosting/custom-domains/dns-configuration)

**Comment by y0geshx** [comment]
does it support vercel or netlify ?

**Reply by wdhdev** [reply]
No

**Comment by JamshaidAhmedd** [comment]
Added _vercel TXT record for Vercel verification. Kindly review. 🙌

**Comment by uzairdeveloper223** [comment]
The is-a.dev now supportes Vercel and netlify , now you can add your is-a.dev domain to vercel or netlify

**Reply by ponderdev** [reply]
what are the benefits though

**Comment by Dallss** [comment]
My PR #27038 was merged almost 2 months ago (Sept 29). But my domain (dallss.is-a.dev) still doesnt seem to exist.

**Reply by STICKnoLOGIC** [reply]
you sure?? 
<img width="1919" height="892" alt="image" src="https://github.com/user-attachments/assets/9688faf4-90b4-402f-bcf1-a6509a3c87bd" />

maybe you have typo error :)

**Reply by Dallss** [reply]
Im sorry it does exist. I used a bad website for checking dns. I've been having trouble making it point to my githup pages deployment despite adding it already as a custom domain.

**Reply by Dallss** [reply]
<img width="1422" height="820" alt="image" src="https://github.com/user-attachments/assets/58f35013-f4c7-4132-a624-d79742552487" />

**Reply by STICKnoLOGIC** [reply]
as I have said "TYPO ERROR" check your PR and see what did you "register" 😊😊

**Reply by Dallss** [reply]
Other than the CNAME's capitalized first letter (Dalls.io), which was fixed on commit d104e639034f97fae8d14cf9f6547720566dbdb8. I don't see any other typo. Im sorry can u point it out more explicitly.

**Reply by STICKnoLOGIC** [reply]
<img width="228" height="47" alt="image" src="https://github.com/user-attachments/assets/defa0a19-2f55-48a5-8ee9-ff69a547d5c5" />


<img width="92" height="34" alt="image" src="https://github.com/user-attachments/assets/28099918-74c0-4ba9-a614-a2dbf0d855ef" />

smh..

**Reply by STICKnoLOGIC** [reply]
are you daydreaming when creating the json file???🤣🤣🤣

**Reply by Dallss** [reply]
oh. Thankyou sm

**Reply by Dallss** [reply]
> are you daydreaming when creating the json file???🤣🤣🤣

This error is so embarassing haha. can i make another pr and tag it here?

**Reply by STICKnoLOGIC** [reply]
no, just create a pr and dont tag anyone or mention here, or else...

**Comment by kakooza-cyber** [comment]
When am I going to get my domain
